src/main/shell-session/node-shell-session/node-shell-session.ts

```diff
@@ -25,7 +25,7 @@ export class NodeShellSession extends ShellSession {
   protected readonly cwd: string | undefined = undefined;
 
   constructor(protected nodeName: string, kubectl: Kubectl, socket: WebSocket, cluster: Cluster, terminalId: string) {
-    super(kubectl, socket, cluster, terminalId);
+    super(kubectl, [], socket, cluster, terminalId);
```

Comment:
Is this correct, that we don't offer modifying the shell env for node shells?

Reply:
I think we should allow it because an extension might need to set some things up for the node to work maybe. but we should pass into the modifiers the type of shell too.

Reply:
This would let an extension set env vars in the terminal that Lens uses solely to launch the `kubectl exec` into the node pod. The terminal and the pod are under Lens's control and are just a means to a shell on the node. Is there something that can be added by an extension to the terminal env that could  positively affect the behaviour of the `kubectl exec` to get the shell?

Being able to set env vars within the node shell might be useful? @jakolehm?

Also, instead of another parameter to the modifiers we could add another method to the main lens extension (i.e. `nodeShellEnvModifier()`) which is of the same modifier type

Reply:
I would focus on "local" terminal envs in this PR. We just need to design the api so that we can introduce node shell envs later (if needed) without breaking the existing api.

Reply:
Ok. As for the terminal that Lens uses solely to launch the `kubectl exec` into the node pod, I don't see a reason to let an extension set its env, which is how this PR is written. Does that make sense?